<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsrFri</class>
 <widget class="QWidget" name="UsrFri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>632</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Adobe 宋体 Std L</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>634</width>
     <height>385</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="LastOUT">
    <item row="2" column="2">
     <widget class="QPushButton" name="ADD_PB">
      <property name="text">
       <string>添加好友</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="m_showMSG">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="m_showMSG_single">
        <property name="font">
         <font>
          <family>宋体</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_shouFriendList">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QListWidget" name="ListWidget">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_sendMSG">
      <property name="text">
       <string>群聊</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_showonlineusrs">
        <property name="text">
         <string>显示在线用户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_flushfries">
        <property name="text">
         <string>刷新好友列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_deletefri">
        <property name="text">
         <string>删除好友</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_searchusrs">
        <property name="text">
         <string>查找用户</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_private_chat">
        <property name="text">
         <string>私聊</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_private_chat_close">
        <property name="text">
         <string>关闭私聊</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_wqz">
        <property name="text">
         <string>五子棋游戏</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="lineEdit_INPUTMESG">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
